### ISSUE TITLE
UI/UX Improvements

### ISSUE THREAD

**Issue by Gitoffthelawn** [odebroqueville/contextSearch#35]
My recommendations:

 - Allow customizing URL for "Search this site with Google" to any URL, and replace the menuitem string with "Search within this site"
 - Move "Options..." to the end of the list

**Comment by odebroqueville** [comment]
Hi,
What do you mean by "Allow customizing URL for "Search this site with Google" to any URL"?
I don't quite understand what you'd like to be able to do!
Re the placing the options to the end of the Context menu, that will be made available in the next release in AMO. You may already test it with the latest version here on Github if you like.

**Comment by Gitoffthelawn** [comment]
The menuitem "Search this site with Google" is hardcoded to search the current site using Google.  What I propose is allowing the user to specify which search engine to use for searching within the current site.

**Comment by odebroqueville** [comment]
The way the search within a website is carried out using Google is by specifying the keyword “site:” followed by the url of the website you want to search. I’m not certain other search engines offer this feature or the keyword might be different for the different search engines. Besides, this wouldn’t make sense for many of the search engines, like Quora, Facebook, LinkedIn, Twitter, Britannica, etc. Obviously, not all of them are true search engines.

**Comment by Gitoffthelawn** [comment]
I agree, although many popular "regular" search engines support the "site:" parameter.

**Comment by modernsoul** [comment]
+1 it will be great to can add search option in the URL as "site:" or other tips as "&tbs=qdr:w" to have the search result for the last week

Thanks

**Comment by Smile4ever** [comment]
In the next version you will be able to move the options menu to the top, bottom or not show it at all.
